firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -333,11 +334,14 @@ private void removeExperiments(Collection<ConditionalUserProperty> experiments)
   }
 
   private void addExperimentToAnalytics(ConditionalUserProperty experiment) {
-    analyticsConnector.setConditionalUserProperty(experiment);
+    AnalyticsConnector connector = this.analyticsConnector.get();
+    if (connector != null) {
```

Comment:
Thinking: this is just a safety check, but `connector` will never be null here (and at the other checks) since `throwAbtExceptionIfAnalyticsIsNull()` is called first in the public methods

Reply:
good point, removed to be consistent with previous behavior